runtime/gc_vlhgc/ConfigurationIncrementalGenerational.cpp

```diff
@@ -272,8 +272,16 @@ MM_ConfigurationIncrementalGenerational::initialize(MM_EnvironmentBase *env)
 
 	bool result = MM_Configuration::initialize(env);
 
+	/* By default disable hot field depth copying */
+	env->disableHotFieldDepthCopy();
+
 	if (result) {
-		extensions->scavengerScanOrdering = MM_GCExtensions::OMR_GC_SCAVENGER_SCANORDERING_BREADTH_FIRST;
+		if (extensions->scavengerScanOrdering != MM_GCExtensions::OMR_GC_SCAVENGER_SCANORDERING_DYNAMIC_BREADTH_FIRST) {
```

Comment:
I don't think we need to change anything here, until we enable dynamic ordering by default.

Reply:
Without this change, won't scan ordering be forced to to breadthFirstScanOrdering, even if the user selects dynamicBreadthFirstScanOrdering?

Reply:
you are probably right. I was probably thinking that command parsing occurs after this, which would be a case with some other early initialization code, but not this one.